app/controllers/books_controller.rb

```diff
@@ -0,0 +1,22 @@
+class BooksController < ApplicationController
+  include DeviseTokenAuth::Concerns::SetUserByToken
+  include Wor::Paginate
+
+  before_action :authenticate_user!
+
+  def index
+    render_paginated Book::Reducer.apply(params), each_serializer: BookPreviewSerializer
+  end
+
+  def show
+    if book
```

Comment:
we try to not use this way and even we try to not use else too hahah
in this case i would do something like this 
```
def show
  return render json: book if book.present?
  head :not_found
end
```